envoyauth/evaluation.go

```diff
@@ -61,43 +63,49 @@ func Eval(ctx context.Context, evalContext EvalContext, input ast.Value, result
 		return err
 	}
 
+	var buf bytes.Buffer
+	ph := topdown.NewPrintHook(&buf)
```

Comment:
Any particular reason we're using a bytes.Buffer here instead of providing a print hook that sends directly to logrus?

Reply:
As I wrote this comment, I realized we should update the plugin to use the logging package in OPA so that the plugin is not directly dependent on logrus.

Reply:
> Any particular reason we're using a bytes.Buffer here instead of providing a print hook that sends directly to logrus?

I'm trying that now -- I had just taken the route of adding to `EvalResult` first, and then it made sense to do it this way. Let's see where the other path goes.

> As I wrote this comment, I realized we should update the plugin to use the logging package in OPA so that the plugin is not directly dependent on logrus.

Agreed, let's make that a follow-up.